### PR TITLE
Release/6.1.0

### PR DESCRIPTION
**Pull request by Raudius**
- ## Release pull request
- [x] Pull latest stable branch `git checkout stableX`
- [x] Branch off a release branch `git checkout -b release/X.X.X`
- [x] Bump versions
  - [x] appinfo/info.xml
  - [x] package.json
  - [x] package-lock.json
- [x] Update changelog
- [x] Commit the version bump and push to origin 
- [ ] Merge release pull request

## Release the app

- [ ] Once merged, checkout the stable branch again `git checkout stableX` and `git pull`
- [ ] Tag a new release `git tag vx.x.x`
- [ ] Push tag
  - [ ] to the main repo: `git push origin stableX --tags`
  - [ ] to the release repo: `git push release stableX --tags`
- [ ] Build release on https://github.com/nextcloud-releases/richdocuments/tags
  - [ ] Draft a new release with the tag, add the changelog and publish
  - [ ] Wait a few minutes for the automation to build the app and to be published to the App Store

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # Changelog
 
-## 6.0.0
+## 6.0.1
```
Comment (juliushaertl) [inline CHANGELOG.md:3]: We should go with 6.1.0 as this is not only a bug fix release ;)

Comment (Raudius) [reply CHANGELOG.md]: Whoops you mentioned this yesterday, I was on autopilot this morning :/

Fixed!

File: appinfo/info.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	<description><![CDATA[This application can connect to a Collabora Online (or other) server (WOPI-like Client). Nextcloud is the WOPI Host. Please read the documentation to learn more about that.
 
 You can also edit your documents off-line with the Collabora Office app from the **[Android](https://play.google.com/store/apps/details?id=com.collabora.libreoffice)** and **[iOS](https://apps.apple.com/us/app/collabora-office/id1440482071)** store.]]></description>
-	<version>6.0.0</version>
+	<version>6.0.1</version>
```
Comment (juliushaertl) [inline appinfo/info.xml:9]:
```suggestion
	<version>6.1.0</version>
```

File: package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
 	"name": "richdocuments",
-	"version": "6.0.0",
+	"version": "6.0.1",
```
Comment (juliushaertl) [inline package-lock.json:3]:
```suggestion
	"version": "6.1.0",
```

File: package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "richdocuments",
 	"description": "Collabora online integration",
-	"version": "6.0.0",
+	"version": "6.0.1",
```
Comment (juliushaertl) [inline package.json:4]:
```suggestion
	"version": "6.1.0",
```

**Review by juliushaertl — COMMENTED** [review]
A few more to adjust 😉

**Pull request closed without merging**